manager/base/src/main/java/org/wildfly/security/manager/JDKSpecific.java

```diff
@@ -17,42 +17,43 @@
  */
 package org.wildfly.security.manager;
 
-import sun.reflect.Reflection;
+import static java.security.AccessController.doPrivileged;
+
+import java.lang.StackWalker.Option;
+import java.security.PrivilegedAction;
+import java.util.List;
+import java.util.stream.Collectors;
 
 /**
- * JDK-specific classes which are replaced for different JDK major versions. This class is for JDK 8.
+ * JDK-specific classes which are replaced for different JDK major versions. This class is for JDK 9.
  * @author <a href="mailto:jucook@redhat.com">Justin Cook</a>
  */
 final class JDKSpecific {
```

Comment:
@cam-rod @fjuma  do we have plans for the `JDKSpecific` classes? Can we refactore them out of the codebase? Since they were introduced to resolve some inconsistencies between Java 8 and Java 9 IIUC

Reply:
I don't think there's a specific requirement for those classes, so likely yes

Reply:
Should I go ahead with refactoring out these classes?